lib/services/users.pm

```diff
@@ -147,6 +147,9 @@ sub full_users_check {
         record_info('Unsupported on non-gnome', "This test is only supported on gnome, quit for your DESKTOP is $desktop", result => 'fail');
         return;
     }
+    # reset consoles before select x11 console will make the connect operation 
+    # more stable on s390x
+    reset_consoles if check_var('ARCH', 's390x');
```

Comment:
Lemon, I suggest set this for all platforms, since reset_consoles can prevent more unstable status.
Given the fact that we hit some issues on s390x, we may hit the issue on other platforms as well.

Reply:
I think it also can improve performance for other platform.

Reply:
> since
Is there any clue same issue happen on other platform? Actually this is workaround here, not official operation of openqa.
s390 using VM client vnc instead of qemu , so some strange issue happen.

Reply:
Good point Wei, 
Reset_consoles:
will make sure the next select_console will activate the console. This is important if you did something to the system that affects the console (e.g. trigger reboot).